readme.md

```
@@ -57,10 +57,12 @@ To use on `$name` param as string.
 | briefcase            | <img align="center" width="30" src="/src/static/nimbus-icons/briefcase.svg">            |
 | browser              | <img align="center" width="30" src="/src/static/nimbus-icons/browser.svg">              |
 | browser-search       | <img align="center" width="30" src="/src/static/nimbus-icons/browser-search.svg">       |
+| calculator           | <img align="center" width="30" src="/src/static/nimbus-icons/calculator.svg">           |
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)